index.html

```diff
@@ -3,12 +3,12 @@
   <head>
     <meta charset="utf8"/>
     <title>No Survival - The Game</title>
-    <link href="/styles/layout.css" rel="stylesheet"/>
```

Comment:
I played around with styles and CSS and forgot to redo my playing around, sorry, adding it again